src/types/TestObjects.ts

```diff
@@ -444,6 +501,33 @@ export class TestSet {
         return transcript[0].channelId
     }
 
+    // Return sourceName of better source (if there is a better one)
+    whichIsBetter(better: string, worse: string, conversationId: string): string | null {
+        const ratingPair = this.getRatingPair(better, worse, conversationId)
+        if (ratingPair) {
+            if (ratingPair.result === RatingResult.FIRST) { 
+                return ratingPair.sourceNames[0]
+            }   
+            else if (ratingPair.result === RatingResult.SECOND) { 
+                return ratingPair.sourceNames[1]
+            }
+        }
+        return null
+    }
+
+    // Update rating pairs to indicate one source is better than another
+    setFirstIsBetter(better: string, worse: string, conversationId: string) {
+        const ratingPair = this.getRatingPair(better, worse, conversationId)
+        if (ratingPair) {
```

Comment:
Is this function named as you intended? the `FirstIsBetter` doesn't seem to make sense. It's not checking which is first or second, it's checking if string is better or worse and then setting the order first or second.

It looks like it's setting `FIRST` on the `better` string.  Seems like `setFirstOnBetter` or `setFirstIfBetter`

Reply:
RatingPair assumes the sourceNames are in alphabetical order.  So when you say the first one is better, whether you set FIRST or SECOND depends on how they sort